Dockerfile

```diff
@@ -31,9 +31,11 @@ ENV DEVKITPPC=/devkitpro/devkitPPC
 # Now we have a container that has the dev environment set up. 
 # Copy current folder into container, then compile
 COPY . /projectroot/
-RUN cd /projectroot && make
+RUN cd /projectroot && make clean && make -j8 package
```

Comment:
`make clean` prevents any local builds (outside of docker) from breaking the docker build. 

`make -j8 package` generates the usbloader_gx zip file and runs with 8 threads